sdk/inc/azure/core/az_span.h

```diff
@@ -80,9 +80,9 @@ AZ_NODISCARD az_span az_span_create(uint8_t* ptr, int32_t size);
  */
 // When updating this macro, also update AZ_PAIR_NULL, which should be using this macro, but can't
 // due to warnings, and so it is using an expansion of this macro instead.
-#define AZ_SPAN_NULL \
-  (az_span) \
-  { \
+#define AZ_SPAN_NULL                       \
```

Comment:
Did you run this across all the files in the repo?

Reply:
All that match `#define.*\\` regex :)